test/lib/ecto_types/schedule_test.exs

```diff
@@ -142,6 +143,52 @@ defmodule Ask.ScheduleTest do
       time = @schedule |> Schedule.next_available_date_time(base)
       assert time == DateTime.from_naive!(~N[2017-10-11 12:00:00], "Etc/UTC")
     end
+
+    test "gets next available time: start date" do
+      # Long before the start date
+      base = DateTime.from_naive!(~N[2017-02-01 00:00:00], "Etc/UTC")
+      time = @schedule |> Schedule.next_available_date_time(base)
+      # The beginning of the first active window after the start date
+      assert time == DateTime.from_naive!(~N[2017-03-05 12:00:00], "Etc/UTC")
+    end
+  end
+
+  describe "last_window_ends_at" do
```

Comment:
To understand that these tests were right took me quite a while
- I had to see that only Sundays and Wednesdays where valid
- I had to go look for a calendar (go back to 2017) and see which days (numbers) where valid or not
- And then I had to realize that the schedule timezone is `gmt-3` and the assert is in `UTC`

For the dates, I would at least express the schedule options in a comment. Like:
```
The valid dates for the survey on October '17 are:
- Sun 1
- Wed 4
- ~Sun 8~ (No because is blocked)
- Wed 11
- Sun 15
```

For the times, can we use only one timezone?

Reply:
Sure! Totally agree! I'll change it! Thanks!!